src/main/scala/centaur/test/CheckFiles.scala

```diff
@@ -26,8 +26,10 @@ case class LocalCheckFiles() extends CheckFiles {
   def checkDirectorySize: String => Int = {s =>
     val d = new java.io.File(s)
     if (d.exists && d.isDirectory)
-      d.listFiles.size
-    else 
+      d.listFiles.length
+    else if (d.exists && d.isFile)
```

Comment:
It seems a little strange that `checkDirectorySize` can be invoked on something that isn't a directory